choose_disk.yml

```diff
@@ -0,0 +1,41 @@
+---
+# Defines deployment design and assigns role to server groups
+
+- hosts: localhost
+  gather_facts: false
+  become: True
+  vars:
+      fsid: "e8ba1fe2-dbf6-4b15-b0ca-2c5ffd97f59e"
+      public_network: 172.16.41.0/24
+      cluster_network: 172.16.41.0/24
+      journal_size: 100
+      journal_collocation: true
+      devices: []
+      cluster: ceph
+      osd_objectstore: filestore
+      osd_group_name: osds
+#      devices: {'storage_disks': {'vendor': '0x1af4', 'ceph_type' : 'data', 'size': 'gte(30 GB)', 'count': 2 }}
```

Comment:
why are these commented out? can they be removed or are they here as an example?